SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/SubJob/SubJobWebRunner/SubJobWebRunner.swift

```diff
@@ -262,14 +262,20 @@ public extension SubJobWebRunning {
     }
 
     func success(actionId: String, actionType: ActionType) async {
+        let isOptOut = actionsHandler?.stepType == .optOut
```

Comment:
I wonder if we should maybe have this as a property/extension of `actionsHandler`? We definitely have a few of these checks on things like `DataBroker` IIRC.